### PR TITLE
CAPO for USDe

### PR DESCRIPTION
**Pull request by Rozengarden**
need test

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,7 @@ common-flags := --legacy --ledger --mnemonic-indexes $(MNEMONIC_INDEX) --sender
 
 # Deploy
 deploy-weeth-mainnet :; forge script scripts/DeployEthereum.s.sol:DeployWeEthEthereum --rpc-url mainnet $(common-flags)
+deploy-weeth-mainnet :; forge script scripts/DeployEthereum.s.sol:DeployUSDeEthereum --rpc-url mainnet $(common-flags)
```
Comment (rustboyar) [inline Makefile:17]: wrong name of the command

Comment (Rozengarden) [reply Makefile]: fixed, thanks

File: scripts/DeployEthereum.s.sol [hunk 2]
```diff
@@ -4,13 +4,16 @@ import {GovV3Helpers} from 'aave-helpers/GovV3Helpers.sol';
...
+import {PriceCapAdapterStable} from '../src/contracts/PriceCapAdapterStable.sol';
+import {IPriceCapAdapter, IChainlinkAggregator} from '../src/interfaces/IPriceCapAdapter.sol';
+import {IPriceCapAdapterStable} from '../src/interfaces/IPriceCapAdapterStable.sol';
 import {WeETHPriceCapAdapter} from '../src/contracts/lst-adapters/WeETHPriceCapAdapter.sol';
 import {OsETHPriceCapAdapter} from '../src/contracts/lst-adapters/OsETHPriceCapAdapter.sol';
 
 library CapAdaptersCodeEthereum {
   address public constant weETH = 0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee;
   address public constant osETH_VAULT_CONTROLLER = 0x2A261e60FB14586B474C208b1B7AC6D0f5000306;
+  address public constant USDe_PRICE_FEED = 0x4c9EDD5852cd905f086C759E8383e09bff1E68B3;
```
Comment (rustboyar) [inline scripts/DeployEthereum.s.sol:16]: this is token address, but should be Chainlink's price feed:
https://data.chain.link/feeds/ethereum/mainnet/usde-usd

Comment (marczeller) [inline scripts/DeployEthereum.s.sol:16]:
```suggestion
  address public constant USDe_PRICE_FEED = 0xa569d910839Ae8865Da8F8e70FfFb0cBA869F961;
```

File: tests/ethereum/USDePriceCapAdapterTest.t.sol [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+contract USDePriceCapAdapterTest is BaseStableTest {
+  constructor()
+    BaseStableTest(
+      CapAdaptersCodeEthereum.USDeAdapterCode(),
+      10,
+      ForkParams({network: 'mainnet', blockNumber: 19940721})
+    )
+  {}
+
+  //function _createAdapter(
```
Comment (rustboyar) [inline tests/ethereum/USDePriceCapAdapterTest.t.sol:17]: you can remove this

Comment (marczeller) [reply tests/ethereum/USDePriceCapAdapterTest.t.sol]: done

**Review by rustboyar — APPROVED** [review]

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**